uiowa/uiowa
# Classroom theming

Resolves #6044 

### Test Instructions:

1. `blt ds --site=classrooms.uiowa.edu`
2. Create a classroom and fill out all the fields available. You may have to create taxonomy terms for the imported fields.
3.  Test the technology details font awesome icons.  Add icons to the details.  Test as webmaster to make sure you don't have access to additional font awesome settings. 
4. Compare with https://sandbox.prod.drupal.uiowa.edu/art-building-west-240-alt. Make sure all the fields are rendering correctly.
5. Remove featured image and see if "Home" breadcrumb is showing up.
6. Test the links in the "Additional Information" section.
